cli/src/semgrep/app/scans.py

```diff
@@ -358,9 +358,6 @@ def report_findings(
             ),
         )
 
-        if self._dependency_query:
-            complete.dependencies = out.CiScanDependencies(lockfile_dependencies)
```

Comment:
can we update semgrep_output_v1.atd and just remove the field there too then?
(removing an optional field is safe; ATD ignores fields he does not know about).

Reply:
Yeah, totally.

Reply:
Well, actually, I think we still support versions that didn't send it to /results and the backend needs to be able to parse it from /complete for now.

Reply:
https://github.com/returntocorp/semgrep-interfaces/pull/148